data/json/items/comestibles.json

```diff
@@ -832,7 +832,8 @@
     "//": "Given the comment, presuming we're taking bourbon here.",
```

Comment:
you missed the wines, which should freeze around 15-20 F, lower for reds that tend to have more alcohol content and higher for some of the whites.